estate/models/estate_property.py

```diff
@@ -5,7 +5,7 @@
 class EstateProperty(models.Model):
     _name = "property"
     _description = "Test Model for Training"
-   
+
     name = fields.Char(string = 'Name', required=True)
     description = fields.Text(string = 'Description')
```

Comment:
No need for "string" here, since is the same as the field name the result will be the same

For example:

"description_for_property" the resulting string will be "Description For Property"

remove the string please

```suggestion
    description = fields.Text()
```